bin/update_ati_analytics.sh

```diff
@@ -78,7 +88,12 @@ if [[ ! -d "${tmp_dir}" ]]; then
 
 fi
 
-cp ${generated_file} ${tmp_file};
+###
+# We do not want the current month's counts
+###
+filter_date="$(date +%Y),$(echo $(date +%m) | sed 's/^0*//'),";
+echo "Filtering out rows: ${filter_date}";
+sed "/^${filter_date}*/d" < ${generated_file} > ${tmp_file};
```

Comment:
sed isn't generally safe for use with csv files. Are we sure there aren't any newlines in the data?

Reply:
Ok, then maybe a `# should be safe, I think there aren't any newlines` comment would be fine

Reply:
Comment added